src/dbus_api/pool.rs

```diff
@@ -3,6 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 use std::collections::HashMap;
+use std::option::Option;
```

Comment:
you are right.... surprised I didn't get a compiler warning.  anyway - I made the change.